AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreConfigurationTests.swift

```diff
@@ -0,0 +1,137 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreConfigurationTests: XCTestCase {
+
+    func testConfigureWithSameSchemaDoesNotDeleteDatabase() throws {
+        Amplify.reset()
```

Comment:
can move this to common setUp method

Reply:
Sorry, do you mean only putting this Amplify.reset() in another function maybe called setup() and use it in these two tests?

Reply:
Hmmm, but is this necessary?

Reply:
the setUp() and tearDown() methods are part of `XCTestCase` and runs for each test method. you don't have to call `setUp()` in your test method and is done automatically for you as part of the test execution. for each test method, it runs setUp(), then your test method, then tearDown(). it is the ideal place to reset the state of the system for your tests

Reply:
Updated